helpers/deploymentConfig.ts

```diff
@@ -914,7 +921,25 @@ export const globalConfig: NetworkConfig = {
             supplyCap: convertToUnit("5000", 18),
             borrowCap: convertToUnit("4000", 18),
             vTokenReceiver: "0xAE1c38847Fb90A13a2a1D7E5552cCD80c62C6508",
-            reduceReservesBlockDelta: "10512000",
+            reduceReservesBlockDelta: "100",
+          },
+          {
+            name: "Venus PLANET (DeFi)",
+            asset: "PLANET",
+            symbol: "vPLANET_DeFi",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: convertToUnit("0.02", 18),
+            multiplierPerYear: convertToUnit("0.2", 18),
+            jumpMultiplierPerYear: convertToUnit("3", 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.2", 18),
+            liquidationThreshold: convertToUnit("0.3", 18),
+            reserveFactor: convertToUnit("0.25", 18),
+            initialSupply: convertToUnit("174983000", 18),
+            supplyCap: convertToUnit("500000000", 18),
```

Comment:
My fault, this should be 1,000,000,000 according to [Chaos labs recommendations](https://community.venus.io/t/proposal-to-list-planet-on-venus/3866/2#recommendations-6)